### DISCUSSION TITLE
Cloud Spoke tag for promotion question...

### DISCUSSION THREAD

**Discussion by pyrate** [mozilla/hubs#3131] [Hubs Cloud Questions] [answered]
Hi all! Cloud Question. In Spoke when you publish a room (via Spoke on our own AWS Cloud instance), and we tag for promotion, that is ONLY for our server correct? Not that it will be submitted to Mozilla's instance for promotion too?

**Comment by robinkwilson** [comment, marked as answer]
Correct. Promotion would be for your own Hubs Cloud instance only.
